server/models/Job.js

```diff
@@ -4,40 +4,40 @@ const { Schema } = mongoose;
 const Proposal = require('./Proposal');
 
 const jobSchema = new Schema({
-    name: {
-        type: String,
-        required: true,
-    },
-    jobCustomer: {
-        type: String,
-        required: true,
-    },
-    description: {
-        type: String,
-        required: true,
-        minLength: 1,
-        maxLength: 1000,
-        trim: true,
-    },
-    skills: {
-
-        required: true,
-    },
-    city: {
-        type: String,
-        required: true,
-    },
-    needDate: {
-        type: Date,
-        required: false,
-    },
-    createdDate: {
-        type: Date,
-        default: Date.now,
-    },
-    proposals: [Proposal.schema]
+  name: {
+    type: String,
+    required: true,
+  },
+  jobCustomer: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+    minLength: 1,
+    maxLength: 1000,
+    trim: true,
+  },
+  skills: {
+    type: [],
+    required: true,
+  },
+  city: {
+    type: String,
+    required: true,
+  },
+  needDate: {
+    type: Date,
+    required: false,
+  },
+  createdDate: {
+    type: Date,
+    default: Date.now,
+  },
+  proposals: [Proposal.schema],
 });
 
 const Job = mongoose.model('Posts', jobSchema);
```

Comment:
I think this 'Posts' is supposed to be Jobs. Which I probably missed when I uploaded it, sorry.

Reply:
Good catch!